### ISSUE TITLE
List plugins in the readme?

### ISSUE THREAD

**Issue by douglasg14b** [dfahlander/Dexie.js#806]
Can 3rd party plugins like dexie-relationships be listed in the readme? Or in a readme under the /addons directory?

It would be very helpful for discovery, and to see if there are any listed known compatibility issues with current versions.

**Comment by dfahlander** [comment]
Yes, we should include them in readme - It's an old todo #465. Currently the place to see related works and addons are here: https://dexie.org/docs/DerivedWork

**Comment by dfahlander** [comment]
I normally give the project full attention in certain periods between customer work only, so some todos may get down-prioritized for a while. A PR or some kind suggestion is always welcome.
